File: 2022_python/220401.py
#
# created by z on 4/1/2022
# 344

class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        n: int = len(s)
        for i in range(n//2):
            s[i], s[-1-i] = s[-1-i], s[i]
        return

File: 2022_04_cpp/220401.cpp
//
// Created by z on 4/1/2022.
// # 344

class Solution {
public:
    void reverseString(vector<char>& s) {
        int n = s.size();
        for (int i = 0; i < n/2; i++)
        {
            char tmp = s[i];
            s[i] = s[n-1-i];
            s[n-1-i] = tmp;
        }
        return;
    }
};

